bandwidth/webrtc/utils/transfer_util.py

```diff
@@ -6,14 +6,14 @@
 @copyright Bandwidth INC
 """
 
-def generate_transfer_bxml(device_token, sip_uri='sip:sipx.webrtc.bandwidth.com:5060'):
+def generate_transfer_bxml(device_token: str, voice_call_id: str, sip_uri: str ='sip:sipx.webrtc.bandwidth.com:5060'):
```

Comment:
Can you add PyDoc for the parameters in each function?

Reply:
Added, thank you!

Reply:
The parameter typing is new as of 3.5 right? Will that potentially break customers on older versions? I don't think we made an explicit python version for this package so we may need to add 3.5 as the lowest supported version

Reply:
@jmulford-bw Google says you're right, I did not think of that. I'm happy to just remove the typing since it's inconsistent with the rest of the sdk anyways.

Reply:
That's totally fine with me

Reply:
Updated, thank you for catching that!